github-pr/mavo-github-pr.js

```diff
@@ -17,7 +136,7 @@
 						head: `${env.user.username}:${env.branch}`,
 						base: env.repoInfo.parent.default_branch
 					}).then(prs => {
-						Mavo.Plugins.loaded["github-pr"].pullRequest.call(env, prs[0]);
+						env.pullRequest(prs[0]);
```

Comment:
Is `env` `this`? If so, I'd suggest using a normal function and using `this` instead of `env`. That way it looks closer to what it would have been if it were not a plugin.

Reply:
`env` is `env.context` originally. I reassigned it to simplify the code.

Reply:
Don't do that. You lose access of all the other properties on `env`. 
The function is automatically called with `env.context` as its context (hence the name) so you can just use `this` (if you're not using an arrow function, which doesn't have its own context).

Reply:
Oh! Sorry. I will fix this now.

Reply:
Fixed! Sent a PR.

TIL the right way of working with hooks. Yay!